scripts/fuzz.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e
+
+function build_fuzz() {
+  PACKAGES=$1
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+  
+    for file in *.go; do
+      if [[ "$file" == 'fuzz_utils.go' ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      echo "Building zip file for $pkg/$NAME"
+      go-fuzz-build -func "Fuzz_$NAME" -o "$pkg-$NAME-fuzz.zip" "github.com/lightningnetwork/lnd/fuzz/$pkg"
+    done
+  
+    popd
+  done
+}
+
+function run_fuzz() {
+  PACKAGES=$1
+  RUN_TIME=$2
+  TIMEOUT=$3
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+    DIR=$(pwd)
+  
+    for file in *.go; do
+      if [[ "$file" == 'fuzz_utils.go' ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      PROCS=$(cat /proc/cpuinfo | grep processor | wc -l)
```

Comment:
macOS: `cat: /proc/cpuinfo: No such file or directory`

Reply:
Okay, made that configurable instead of auto-detected.